data/time_series.go

```diff
@@ -181,9 +253,22 @@ func LongToWide(longFrame *Frame) (*Frame, error) {
 				longField := longFrame.Fields[tsSchema.ValueIndices[offset]]
 
 				newWideField := NewFieldFromFieldType(longField.Type(), wideFrameRowCounter+1)
+				if fillMissing != nil && fillMissing.Mode != FillModeValue {
+					// if fillMissing mode is null or previous
+					// the new wide field should be nullable
+					// because some cells can be null
+					newWideField = NewNullableFieldFromFieldType(longField.Type(), wideFrameRowCounter+1)
+				}
 				newWideField.Name, newWideField.Labels = longField.Name, labels
 				wideFrame.Fields = append(wideFrame.Fields, newWideField)
 
+				fillVal, err := getMissing(fillMissing, newWideField, wideFrameRowCounter)
+				if err == nil {
```

Comment:
Just checking, we should suppress any error here, i.e. not propagate it?

Reply:
See my comment [here](https://github.com/grafana/grafana-plugin-sdk-go/pull/135/files#r408645997).